lib/local-links-manager/export/bad_links_url_and_status_exporter.rb

```diff
@@ -3,22 +3,22 @@
 module LocalLinksManager
   module Export
     class BadLinksUrlAndStatusExporter
-      HEADINGS = %w(url status).freeze
-
+      HEADINGS = %w(url link_errors link_warnings).freeze
       def self.local_authority_bad_homepage_url_and_status_csv
         CSV.generate do |csv|
           csv << HEADINGS
-          LocalAuthority.where.not(status: "200").distinct.pluck(:homepage_url, :status).each do |la|
-            csv << la
+          LocalAuthority.where.not(status: "ok").pluck(:homepage_url, :link_errors, :link_warnings).each do |la|
+            csv << [la[0], la[1].join(','), la[2].join(',')]
```

Comment:
not sure about the wisdom of using comma separators here since we have commas in some of the messages. I wonder if we could just have status and problem summary and not worry about the lists?